### PR TITLE
Add support for HTTP form parsing through a vendored copy of Werkzeug

### PR DESCRIPTION
**Pull request by elprans**
Closes: Azure/azure-functions-python-worker#349

### CODE REVIEW

**Comment by asavaritayal** [comment]
@maiqbal11 can you please provide a review?

**Comment by maiqbal11** [comment]
> @maiqbal11 can you please provide a review?

Had reviewed a few days ago but forgot to submit. Added now.

File: azure/functions/_http.py [hunk 1]
```diff
@@ -177,8 +185,41 @@ def params(self):
...
+        return self.__files
+
     def get_body(self) -> bytes:
         return self.__body_bytes
 
     def get_json(self) -> typing.Any:
         return json.loads(self.__body_bytes.decode())
+
+    def _parse_form_data(self):
+        if self.__form_parsed:
```
[the remark this answers is not in the document]
Comment (elprans) [reply azure/functions/_http.py]: Fixed. Thanks.

File: tests/test_http.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import unittest
+
+import azure.functions as func
+
+
+class TestHTTP(unittest.TestCase):
+    def test_http_form_parse_01(self):
```
[the remark this answers is not in the document]
Comment (elprans) [reply tests/test_http.py]: Amended.

**Pull request closed without merging**